src/ingest_llm_as/api/repository.py

```
@@ -71,6 +81,45 @@ async def ingest_python_repository(
+            source_type="python-repo",
+            content_type="repository",
+            raw_payload=request.model_dump(mode="json"),
+            raw_metadata={
+                "repository_source": request.repository_source.value,
+                "source_path": request.source_path,
+                "max_files": request.max_files,
+                "max_file_size": request.max_file_size,
+                "include_patterns": request.include_patterns,
+                "exclude_patterns": request.exclude_patterns[:5],  # First 5 for brevity
+            },
+            captured_at=datetime.utcnow(),
```

Ruff (0.14.11)

`datetime.datetime.utcnow()` used

(DTZ003)